qcodes/instrument_drivers/Harvard/Decadac.py

```diff
@@ -141,7 +188,7 @@ def _setvoltage(self, voltage, channel):
             mssg += ''.join(script) + runcmd
             self.visa_handle.write(mssg)
             sleep(0.0015*self.ramp_time)  # Required sleep.
```

Comment:
whattttt?? I changed it but same error message....


--> 186             sleep(0.0015*self._ramp_time)  # Required sleep.
    187             self.visa_handle.read()
    188 

a:\qcodes\qcodes\utils\helpers.py in __getattr__(self, key)
    320         raise AttributeError(
    321             "'{}' object and its delegates have no attribute '{}'".format(
--> 322                 self.__class__.__name__, key))
    323 
    324     def __dir__(self):

AttributeError: ("'Decadac' object and its delegates have no attribute 'ramp_time'", 'setting Decadac:ch0_voltage to 0')